Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ruby:alpine
+WORKDIR /code
+COPY . /code
+RUN apk update && apk add --virtual build-dependencies build-base ruby-dev libpq postgresql-dev tzdata
```

Comment:
Please run apk update and apk add  at the begging of the Dockerfile. That way it'll get cached.